pyasdf/generic_io.py

```diff
@@ -531,19 +531,23 @@ def read_until(self, delimiter, delimiter_name=None, include=True):
         raise ValueError("{0} not found".format(delimiter_name))
 
     def seek_until(self, delimiter, include=True):
+        last_block = b''
         while True:
             block = self.read_block()
             if not len(block):
                 break
-            index = re.search(delimiter, block)
+            blocks = last_block + block
+            index = re.search(delimiter, blocks)
```

Comment:
Nevermind, I see now why that wouldn't work.  Though I guess it could also do
```python
index = re.search(delimiter, last_block)
if index is None:
    index = re.search(delimiter, block) + len(last_block)
```
and still avoid the concatenate.  I don't know how large these blocks are or if it matters though.